docs/patterns/components/Accordion/index.js

```diff
@@ -71,15 +71,14 @@ const AccordionDemo = () => {
   AccordionContent,
 } from '@deque/cauldron-react/';
 
-const ControlledAccordion = ({ label }) => {
-  const [open, setIsOpen] = useState(false);
+const ControlledAccordion = ({ label, open: initialOpen = false }) => {
+  const [open, setIsOpen] = useState(initialOpen);
 
   return (
     <Accordion open={open} onToggle={() => setIsOpen(!open)}>
-    <Accordion>
-    <AccordionTrigger heading={{ level: 4 }}>
-      {label}
-    </AccordionTrigger>
+      <AccordionTrigger heading={{ level: 4 }}>
+        {label}
+      </AccordionTrigger>
     <AccordionContent>Here is some content</AccordionContent>
   </Accordion>
```

Comment:
nit: Some of the alignment on this is off:

![image](https://user-images.githubusercontent.com/6416791/181103251-0fbe2876-0781-4386-bbd8-e64ae4c05927.png)

Reply:
I really hate our documentation here. I wish we used markdown or something we could run prettier over.